src/storm/solver/helper/ValueIterationOperator.cpp

```diff
@@ -3,6 +3,8 @@
 #include <optional>
 
 #include "storm/adapters/RationalNumberAdapter.h"
+#include "storm/adapters/RationalNumberForward.h"
```

Comment:
If we already include the adapter, then there should be no need to inlcude the forward